lib.rs

```
@@ -105,19 +99,25 @@ pub fn with_name<W>(name: &'static str, io: W) -> slog_json::JsonBuilder<W>
 where
     W: io::Write,
 {
-    new_with_ts_fn(name, io, |_: &Record| chrono::Local::now().to_rfc3339())
+    new_with_ts_fn(name, io, |_: &Record| {
+        time::OffsetDateTime::now_local()
+            .unwrap_or_else(|_| time::OffsetDateTime::now_utc())
+            .format(&time::format_description::well_known::Rfc3339)
+            .ok()
```

Why does this implicitly suppress errors here?

Are there alternatives?

---

So looking at the time documentation, formatting errors can only happen if:
1. There is [`InsufficientTypeInformation`](https://docs.rs/time/latest/time/error/enum.Format.html#variant.InsufficientTypeInformation)
2. There is [`InvalidComponent`](https://docs.rs/time/latest/time/error/enum.Format.html#variant.InvalidComponent) 
3. There is an internal `std::io::Error`

None of these are really concerns for us 😉